packages/core/src/rum/instrumentation/DdRumErrorTracking.tsx

```diff
@@ -100,17 +103,36 @@ export class DdRumErrorTracking {
 
     }
 
+    private static getErrorMessage(error: any | undefined): string {
```

Comment:
seems `undefined` is redundant here, because `onGlobalError` has `error: any`, without `undefined`. `onGlobalError` [is called](https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/packages/polyfills/error-guard.js#L44-L50) with `mixed` type in JS, which seems counterpart of `any` and I guess doesn't include `undefined`.

Reply:
true

Reply:
Actually it's also used from the onConsoleError with params of type `unknown`, so better safe than sorry.